apps/cowswap-frontend/src/legacy/state/application/hooks.ts

```diff
@@ -88,10 +56,6 @@ export function useActivePopups(): AppState['application']['popupList'] {
   return useMemo(() => list.filter((item) => item.show), [list])
 }
 
-export function useToggleTransactionConfirmation(): () => void {
-  return useToggleModal(ApplicationModal.TRANSACTION_CONFIRMATION)
-}
-
 // TODO: These two seem to be gone from original. Check whether they have been replaced
```

Comment:
With the new modals, can this part be killed too? (the legacy redux state modals)

Reply:
Unfortunately no, they are still in use